src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,49 @@
+.nav {
+  &__list {
+    display: flex;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__item {
+    list-style: none;
+    position: relative;
+    margin-left: 20px;
+    height: 60px;
+
+    &:first-child {
+      margin-left: 0;
+    }
+  }
+
+  &__link {
+    color: #060b35;
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    line-height: 60px;
+    font-weight: 500;
+    text-decoration: none;
+    white-space: nowrap;
+    display: block;
+    transition-duration: $trans-duration;
```

Comment:
The variable `$trans-duration` is used but not defined in this file. Ensure that it is defined elsewhere in your stylesheets or consider defining it here.